packs/golden-signals-web/config.yml

```diff
@@ -0,0 +1,19 @@
+
+# Name of the pack
+name: Golden Signals for Web Servers
+
+# Description of the pack
+description: |+
+  Golden signals alerts for web servers. Includes alerts and a dashboard for throughput, errors, response time, CPU usage, and memory usage.
+
+# Support level: New Relic | Verified | Community
+level: New Relic
+
+# Design
+icon: icon.jpeg
```

Comment:
I guess there isn't really a better icon/logo for this, so I am ok with the New Relic logo